libs/template/templates/lakeflow-pipelines/template/{{.project_name}}/databricks.yml.tmpl

```
@@ -1,12 +1,12 @@
 # This is a Databricks asset bundle definition for {{.project_name}}.
 # See https://docs.databricks.com/dev-tools/bundles/index.html for documentation.
-bundle:
+{{$isDLT := eq .is_dlt true}}{{if $isDLT}}project{{else}}bundle{{end}}:
```

```suggestion
{{- $isDLT := eq .is_dlt true}}
{{if $isDLT}}project{{else}}bundle{{end}}:
```

Please play a bit with the `-` here to make this a bit easier to read. The `-` chops a way all whitespace.